### DISCUSSION TITLE
Paying for Linux Build

### DISCUSSION THREAD

**Discussion by pleasehelp523** [sorayuki/obs-multi-rtmp#231] [General]
I will pay the creator of this OBS plugin $300 for a working Linux version.

I see a Linux build script, but I can't run it due to difficulty in finding the QT and other dependency files required.

**Comment by sorayuki** [comment]
Flatpak is not working in my computer. I use the one installed by apt on Ubuntu as example.

1. install obs-studio by apt

2. install dependencies.
```
sudo apt install build-essential cmake mesa-common-dev qt6-base-dev
```

3. download source code of this project (or git clone)

4. in terminal, switch to the source code's directory and execute
```
cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
cmake --build build
sudo cmake --install build
```

5. the multi-rtmp dock seems working incorrectly to attach to the main GUI. Maybe fixed some time.

**Reply by franpavonvon** [reply]
Hi, I use MX-Linux. I have this error:

CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "FindQt6.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt6", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt6" with any of
  the following names:

    Qt6Config.cmake
    qt6-config.cmake

  Add the installation prefix of "Qt6" to CMAKE_PREFIX_PATH or set "Qt6_DIR"
  to a directory containing one of the above files.  If "Qt6" provides a
  separate development package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
See also "/home/fran/obs-multi-rtmp/build/CMakeFiles/CMakeOutput.log".
gmake: *** No se especificó ningún objetivo y no se encontró ningún makefile.  Alto.
CMake Error: Error processing file: /home/fran/obs-multi-rtmp/build/cmake_install.cmake

**Comment by AntonioGomes42** [comment]
I made a new tutorial showing how to install it on a Flatpak OBS. I works for every single linux. Link : https://youtu.be/p4eoEoq438M. I hope it helps you!

**Comment by sxnvmqe** [comment]
https://github.com/sxnvmqe/obs-multi-rtmp-flatpak
